### PR TITLE
Bump to 1.5.4, bump 'Tested up to' version to 5.4

### LINKED ISSUE #519: Ensure compatibility with WordPress Core 5.4
## Feature description
Test all of the controls to ensure the look good and work the same in WP 5.4

## Additional information
1. Full regression testing for WP 5.4
2. Bump the 'Tested up to' version to 5.4

### PR DESCRIPTION
**Pull request by kienstra**
#### Changes
* Bump BL version and 'Tested up to' version
* Add `CHANGELOG.md` entries

#### Testing instructions
* Regression testing already done for WP 5.4 in https://github.com/getblocklab/block-lab/issues/519#issuecomment-604214696
* A little more regression testing on 5.3.2 would be nice, though I did quick testing on this

Closes #519

### CODE REVIEW

**Comment by kienstra** [comment]
For testing, here's the [diff](https://github.com/getblocklab/block-lab/compare/b8011ccfa4782710ba814489af4ebebc6713598f...73c38e2f68a951dbfad71df30ff622016002e85b) since the last [version bump PR](https://github.com/getblocklab/block-lab/pull/502).

**Comment by kienstra** [comment]
Hi @lukecarbis and @RobStino,
Could you please review this?

I [couldn't find issues](https://github.com/getblocklab/block-lab/issues/519#issuecomment-604214696) with the upcoming WP 5.4. But it'd be great if you could test that briefly and WP 5.3.2.

We have a little time until the planned March 31st [WP 5.4 release date](https://wordpress.org/news/2020/03/wordpress-5-4-release-candidate/), but a lot of that is around the weekend.

Thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 ## Changelog #
 
+### 1.5.4 – 2020-03-26 ###
```
Comment (kienstra) [inline CHANGELOG.md:3]: Of course, we could change this date, and it could refer to Australian or North American time zones

**Comment by kienstra** [comment]
Here's a build of this branch:

[block-lab.zip](https://github.com/getblocklab/block-lab/files/4396430/block-lab.zip)

**Comment by kienstra** [comment]
Hi @lukecarbis,
Hope you're doing well.

Could you please review this? We need to release this before WP 5.4, which is planned for 2 days from now. I could take care of the deployment to wp.org if you'd like.

Thanks, Luke!

**Review by lukecarbis — APPROVED** [review]

**Comment by kienstra** [comment]
Nice, thanks for reviewing and merging!

**Pull request merged**